source/index.txt

```diff
@@ -1,14 +1,34 @@
 .. _c2c-index:
 
-===============================
-MongoDB Cluster-to-Cluster Sync
-===============================
+=======================
+Cluster-to-Cluster Sync
+=======================
 
 .. default-domain:: mongodb
 
-The primary motivation of this initiative ...
+The :ref:`c2c-mongosync` tool connects MongoDB clusters so data can
+flow between them.
 
-NOTE: This page will have material that introduces C2C to the reader.
+``mongosync`` provides continuous cluster to cluster data
+synchronization. You can use ``mongosync`` to create dedicated
+analytics or testing environments. Synchronized clusters can also
+support audit and data residency requirements. 
+
+In addition to continuous data synchronization, ``mongosync`` can also
+facilitate a one time data migration between clusters. 
+
```

Comment:
In the paragraph above, can we clean up the structure?

mongosync can provides you with continuous sync (indefinite) or one-time data migration between clusters. 

mongosync allows continuously sync data between two clusters, enabling you to create separate development/test environments, create dedicated analytics environments, and meet locality requirements for auditing and compliance.

In addition to continuous data synchronization, mongosync can also facilitate a one-time data migration between clusters.